.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,39 +1,48 @@
 ---
 name: Bug Report
 about: Template to report bugs
-title: ''
+title: 'bug: '
 labels: ''
 assignees: ''
 
 ---
 
-Description of the problem:
+# Bug Report
+
+## Capacitor Version
```

Comment:
maybe tell them to run `capacitor doctor` and paste the whole `Installed Dependencies:` section, as capacitor is made by a lot of packages and they might have the CLI updated but not the platforms

Reply:
??? `capacitor doctor`? Is that a thing?

Reply:
yeah, but should be executed with npx ( `npx cap doctor` or `npx capacitor doctor`), I forgot I have a global capacitor for testing, but for regular users it won't work without npx